test/validators/isMobilePhone.test.js

```diff
@@ -0,0 +1,102 @@
+import test from '../testFunctions';
+
+describe('isMobilePhone', () => {
```

Comment:
Please add this to the big validators test file, where the other isMobilePhone tests are.